amalgamation/amalgamation.py

```diff
@@ -143,7 +143,12 @@ def expand(x, pending, stage):
                 continue
             path = m.groups()[0]
             h = path.strip('./') if "../3rdparty/" not in path else path
-            source = find_source(h, x, stage)
+            if h.endswith('complex.h') and x.endswith('openblas_config.h'):
+                source = ''
+            elif h.startswith('ps/'):
```

Comment:
Where does the "ps/" come from? Maybe we can edit where this comes from instead of adding an exception?

Reply:
"ps/" come from

https://github.com/apache/incubator-mxnet/blob/4649bfa641ad4129b3e83ea0af14b489e512f8f4/src/kvstore/kvstore_dist.h#L33

Reply:
Feel free to rewrite that instead. It doesn't seem correct to me.

@szha @eric-haibin-lin